packages/pds/src/auth-verifier.ts

```diff
@@ -253,28 +254,28 @@ export class AuthVerifier {
     }
   }
 
-  adminService = async (reqCtx: ReqCtx): Promise<AdminServiceOutput> => {
-    if (!this.dids.admin) {
+  modService = async (reqCtx: ReqCtx): Promise<ModServiceOutput> => {
+    if (!this.dids.modService) {
       throw new AuthRequiredError('Untrusted issuer', 'UntrustedIss')
     }
     const payload = await this.verifyServiceJwt(reqCtx, {
       aud: this.dids.entryway ?? this.dids.pds,
-      iss: [this.dids.admin],
+      iss: [this.dids.modService, `${this.dids.modService}#atproto_mod`],
```

Comment:
We might consider using the service id rather than the key id for the issuer here.

Reply:
yup done 👍